routes/dashboard/api/todo.ts

```diff
@@ -12,9 +12,10 @@ export const handler: Handlers = {
 
       return Response.json(null, { status: 201 });
     } catch (error) {
+      console.log({ error });
       const status = error instanceof AuthError ? 401 : 400;
 
-      return new Response(error, { status });
+      return new Response(error.message, { status });
```

Comment:
Nice. Since this is an API, would it be better to return the error as JSON via `Response.json(error, { status })` instead? WDYT?

Reply:
No strong feelings on this 👍

Reply:
On 2nd thought, this is fine.